src/jalr.cpp

```diff
@@ -25,21 +25,18 @@ void JALR::execute() {
 	if (!func3){
 	uint32_t target = (regs->read(rs1) + imm) & TARMASK;
 		if ((target & ALIGNMASK) != 0){
-			//exception == true
-			#ifdef DEBUG
-			printf("Misaligned memory reference in JALR instruction");
-			#endif
-			return;
+			printf("Misaligned memory reference in JALR instruction, you've activated my trap card!\n");
+			exit(1);
 		}
 	regs->write(rd, regs->readPC());
 	regs->updatePC(target);
 	#ifdef DEBUG
 	printf("Writing 0x%8X to register %d, new PC address is 0x%8X. RS1(%d) value was 0x%8X\n", regs->readPC(), rd, target, rs1, regs->read(rs1));
 	#endif
 		if(rs1 == ra && target == 0 && rd == zero){
-			//#ifdef DEBUG
+			#ifdef #DEBUG
```

Comment:
Remove # in front of DEBUG
```suggestion
			#ifdef DEBUG
```